gapis/gfxapi/vulkan/vulkan.api

```diff
@@ -4607,6 +4607,11 @@ cmd VkResult vkResetDescriptorPool(
     VkDevice                   device,
     VkDescriptorPool           descriptorPool,
     VkDescriptorPoolResetFlags flags) {
+  sets := DescriptorPools[descriptorPool].DescriptorSets
```

Comment:
If I understand it correctly, this should create a copy, so we won't erasing items while iterating them.